bin/pa11y.js

```diff
@@ -117,7 +118,8 @@ function handleHelp() {
  */
 async function runProgram() {
 	if (program.environment) {
-		outputEnvironmentInfo();
+		const info = await outputEnvironmentInfo();
```

Comment:
Can you add the `console.log` to `outputEnvironmentInfo` rather than having it in here.